lib/rules/function-url-scheme-blacklist/README.md

```diff
@@ -0,0 +1,63 @@
+# function-url-scheme-blacklist
+
+Specify a blacklist of disallowed URL schemes.
+
+```css
+a { background-image: url('http://www.example.com/file.jpg'); }
+/**                        ↑
+ *           This URL scheme */
+```
+
+A [URL scheme](https://url.spec.whatwg.org/#syntax-url-scheme) consists of alphanumeric, `+`, `-`, and `.` characters. It can appear at the start of a URL and is followed by `:`.
+
+This rule treats URL schemes as case insensitive (`https` and `HTTPS` are the same).
```

Comment:
Yes. I've added tests to show this.

Like all the other `*-*list` rules, this rule now _only_ accepts lowercase primary options but is case insensitive _when checking for matches_.

So `function-url-scheme-blacklist: "http"` will disallow both `http://g.co/x.jpg` and `HTTP://g.co/x.jpg`.

https://github.com/stylelint/stylelint/issues/2660 is about allowing the user to specify primary options in _any case_ e.g. `function-url-scheme-blacklist: "HTTP"`, or `function-blacklist: "translateX"`.